_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -873,6 +873,9 @@ Gruntwork AWS Landing Zone solution.
 [[pre_requisites]]
 === Pre-requisites
 
+# TODO: one of the pre-requisites should DEFINITELY be, to create infra-live and infra-modules repos, and to pay careful attention
```

Comment:
Good point. 

Note: we haven't updated this guide for the Service Catalog, which will only need infra-live.